bin/ghe-restore

```diff
@@ -104,7 +104,7 @@ cleanup () {
     update_restore_status "$1"
   fi
 
-  if ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
+  if ACTIONS_STOPPED && ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
```

Comment:
I thought about trying to remove the `ghe-actions-start` step from `cleanup` and add a special `cleanup` method that is only `trap`ed after we've stopped actions but this seemed like the saner solution.